data/StevenBlack/hosts

```diff
@@ -1084,9 +1084,6 @@
 0.0.0.0 www.sebder.roboinside.me
 0.0.0.0 www.roboinside.me
 0.0.0.0 robux.dev
-0.0.0.0 rollbar.com
-0.0.0.0 api.rollbar.com
-0.0.0.0 www.rollbar.com
```

Comment:
If the error logging might be considered tracking, then we should at least let the website itself be accessible and remove only
```
0.0.0.0 api.rollbar.com
```

Reply:
Indeed!